applications/machine_learning/src/modules/ml_runner.c

```diff
@@ -202,8 +202,15 @@ static bool handle_sensor_event(const struct sensor_event *event)
 		return false;
 	}
 
-	int err = ei_wrapper_add_data(sensor_event_get_data_ptr(event),
-				      sensor_event_get_data_cnt(event));
+	size_t data_cnt = sensor_event_get_data_cnt(event);
+	float float_data[data_cnt];
+	const struct sensor_value *data_ptr = sensor_event_get_data_ptr(event);
+
+	for (size_t i = 0; i < data_cnt; i++) {
+		float_data[i] = sensor_value_to_double(&data_ptr[i]);
```

Comment:
Do not understand. We add same amount of data as previously. We only need to convert it to float.

Reply:
It would be possible to split sensor_event data into several smaller packages but I do not see a point in it. It is not so much data. Currently when we use acetometer it uses 3x4 B + 3x8 B. Previously it was  3x4 B. still is 24B on stack so much we should worry about it?